src/reconstruct/helper.ts

```diff
@@ -1,20 +1,69 @@
-import { NodeId, ParentInformation, RNode, RType } from '../r-bridge';
-import { ReconstructionConfiguration } from './reconstruct';
+import { NodeId, ParentInformation, RNode, RType } from '../r-bridge'
+import { ReconstructionConfiguration } from './reconstruct'
 
 
 /*
 --helper function--
 */
 export type Selection = Set<NodeId>
+interface PrettyPrintLinePart {
+	part:   string
+	spaces: number
+}
 export interface PrettyPrintLine {
-	line: string
+	line:   string
 	indent: number
 }
-export function plain(text: string): PrettyPrintLine[] {
-	return [{ line: text, indent: 0 }]
+export interface WrappedPrettyPrintLine { //change name to be more fitting
+	linePart: PrettyPrintLinePart[]
+	indent:   number
+}
+export function plain(text: string,space: number): Code {
+	return [unwrapPrettyPrintLine({linePart: [{part: text,spaces: space}],indent: 0})]
```

Comment:
I am not too sure if we should unwrap here already, in my mind, pretty-print lines should remain as long as possible (until they are all converted all at once in the end.